instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py

```diff
@@ -1069,18 +1069,6 @@ def test_uninstrument_after_instrument(self):
         spans = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(spans), 3)
 
-    def test_no_op_tracer_provider(self):
```

Comment:
Why remove this test?

Reply:
> Why remove this test?

Hi, emdneto. I changed the UT code locally and this part is added after I made change. Yes, you're right. This test should keep. I will update my branch to add it. Thanks for your review.